Cargo.toml

```diff
@@ -33,10 +33,14 @@ default = [
   "mio-uds",
   "num_cpus",
   "pin-project-lite",
+  "serde-support",
 ]
 docs = ["attributes", "unstable"]
 unstable = ["default", "broadcaster"]
 attributes = ["async-attributes"]
+serde-support = [
+  "serde",
+]
```

Comment:
I think this feature might need to depend on `"default"`, as the `path` submodule not available on `"std"`. That way people don't run into any surprises.

Reply:
Added!

I think this could be relaxed, if there are serializable types that are present in non-`std` configurations. That should be possible to do down the road though.